src/edu/csus/ecs/pc2/core/strategies/MaxSubmissionsPerMinuteStrategy.java

```diff
@@ -91,12 +89,18 @@ private int getNumberOfRunsSubmittedInTheLastMinute(IInternalContest contest, Ru
                 if (runSubmisionTimeSecs >= contestElapsedSecs-60) {
                     
                     //the run was submitted in the last minute; count it
-                    numSubmittedInLastMinute++ ;
+                    countSoFar++ ;
+                    
+                    //if the submitter has already exceeded the limit, no need to check any more runs
+                    if (countSoFar > maxPerMinute) {
```

Comment:
Shouldn't this be if countSoFar >= maxPerMinute ?

Reply:
Yes; pushed a fix for this.   This was a side-effect of converting the method from `int getNumberOfSubmissions()` to `boolean getIfLimitExceeded()`; previously it was counting the last-found one; now it should be.  Fixed.

I also bumped the current limit-per-minute to 6.  If you approve these additional commits, this is ready to be merged.